Popup/Popup.js

```diff
@@ -58,42 +75,141 @@ const PopupBase = kind({
 		 */
 		centered: PropTypes.bool,
 
+		/**
+		 * When true, popup displays a close button.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
 		closeButton: PropTypes.bool,
+
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal elements and states of this component.
+		 *
+		 * The following classes are supported:
+		 *
+		 * * `popup` - The root class name
+		 * * `body` - Applied to the body content container
+		 * * `popupTransitionContainer` - Applied to the Popup's outermost container. Sizing can be
+		 *                                applied here for percentage-of-screen values.
+		 * * `top` - Applied when the `position` is 'top'
+		 * * `right` - Applied when the `position` is 'right'
+		 * * `bottom` - Applied when the `position` is 'bottom'
+		 * * `left` - Applied when the `position` is 'left'
+		 *
+		 * @type {Object}
+		 * @public
+		 */
 		css: PropTypes.object,
+
+		/**
+		 * Controls how long the transition should take.
+		 * Supported preset durations are: `'short'` (250ms), `'medium'` (500ms), and `'long'` (1s).
+		 * `'medium'` (500ms) is default when no others are specified.
```

Comment:
I'm sorry you need to update this part as well.

Reply:
It's ok, I'm sorry for not noticing. Thanks!!